php/src/Types.cpp

```diff
@@ -1850,7 +1868,7 @@ IcePHP::SequenceInfo::unmarshalPrimitiveSequence(
                 // TODO: can we remove this now
```

Comment:
Can we remove this now?

Reply:
I think we should remove the TODO, one 32 bit platform still in use is ARMHF

Reply:
Keep in mind this is about the PHP numeric type, not the C++ int64_t